### PR TITLE
453-readme-added contribution documentation guidelines to improve the…

### PR DESCRIPTION
**Pull request by ruloCode**
Closes # 453

## 📝 Description

> Added contribution guidelines and update readme.md

It is a proposal to standardize the naming of branches and commits in addition to define the pull request process

## ✅ Pull Request Checklist:

- [ ] :ok_hand: Design updates are approved by design team
- [ ] :white_check_mark: Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] :wheelchair: Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,11 +1,77 @@
...
+
+### Commits naming standard
+
+[Jira task id] + [Package] [Verb] + [Substantive] + [What for]
+
+### Example
+
+```bash
+git commit -m '001-layout-Added flex layout for managing out templates'
+```
```
Comment (DanielSepulveda) [inline CONTRIBUTING.md:32]: Esto no es correcto porque nosotros utilizamos la convencion de [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,11 +1,77 @@
...
+4- Submit a GitHub Pull Request and Get someone else to Code Review.
+
+5- After approved and all Circle CI checks have passed you can now merge the branch. Make sure you only have the commits you want to go into master
+
+```bash
+$ git checkout master
+$ git pull
+$ git merge <branch-name> --ff-only
+$ git push
+```
```
Comment (DanielSepulveda) [inline CONTRIBUTING.md:77]: Esto no es correcto porque no usamos CircleCI y tampoco debemos de hacer los merges asi, la convencion hasta ahorita es que los merges se hagan por github utilizando el modo `squash and merge`

Comment (ruloCode) [reply CONTRIBUTING.md]: corregido

File: CONTRIBUTING.md (hunk 1 above)
Comment (ruloCode) [reply CONTRIBUTING.md]: corregido

**Pull request closed without merging**